### PR TITLE
Support use list

### PR DESCRIPTION
**Pull request by tobixx**
Forwards to use_list param of msgpack.Unpacker

### CODE REVIEW

File: txmsgpackrpc/protocol.py [hunk 1]
```diff
@@ -506,9 +512,11 @@ def __init__(self, group, ttl, port=None, timeout=30, handler=None, sendErrors=F
         @type packerEncoding: C{str}
         @param unpackerEncoding: encoding used for decoding msgpack bytes. Default is 'utf-8'.
         @type unpackerEncoding: C{str}.
+        @param useList: If true, unpack msgpack array to Python list.  Otherwise, unpack to Python tuple.
+        @type useList: C{bool}.
         """
         super(MsgpackMulticastDatagramProtocol, self).__init__(handler=handler, timeout=timeout, sendErrors=sendErrors,
-            packerEncoding=packerEncoding, unpackerEncoding=unpackerEncoding)
+            packerEncoding=packerEncoding, unpackerEncoding=unpackerEncoding, useList=True)
```
Comment (jakm) [inline txmsgpackrpc/protocol.py]: suppose this should be useList=useList

**Review by jakm — CHANGES_REQUESTED** [review]

File: txmsgpackrpc/protocol.py (hunk 1 above)
Comment (tobixx) [reply txmsgpackrpc/protocol.py]: Sure, good catch. Updated.

**Comment by jakm** [comment]
please rebase over actual master, I've moved tests because of some weird behaviour of trial in travis-ci

**Comment by tobixx** [comment]
Rebase done.

**Review by jakm — APPROVED** [review]

**Comment by jakm** [comment]
nice work, thanks

**Pull request merged**